app/Composers/StatusPageComposer.php

```diff
@@ -62,9 +64,100 @@ public function compose(View $view)
         $componentGroups = ComponentGroup::whereIn('id', $usedComponentGroups)->orderBy('order')->get();
         $ungroupedComponents = Component::enabled()->where('group_id', 0)->orderBy('order')->orderBy('created_at')->get();
 
+        // Downtime stats for uptime table
+        $availability = array();
+
+
+        $now = Carbon::now();
```

Comment:
If you say $startOfDay = Carbon::now(); and $endOfDay = Carbon::now(); you can avoid this line.